osbs/repo_utils.py

```diff
@@ -239,6 +239,8 @@ def __init__(self, dir_path='', file_name=ADDITIONAL_TAGS_FILE, tags=None):
         self._from_container_yaml = True if tags else False
         self._file_path = os.path.join(dir_path, file_name)
 
+        if os.path.exists(self._file_path):
+            logger.warning('%s file is deprecated, use tags in container.yaml instead', file_name)
```

Comment:
Could we be slightly more emphatic here to say ".. is deprecated and will no longer be supported in a future version. Please consider using the tags list in ...."?

Reply:
updated